homeassistant/components/onewire/onewirehub.py

```diff
@@ -66,13 +72,29 @@ def _discover_devices_owserver(self):
         """Discover all owserver devices."""
         devices = []
         for device_path in self.owproxy.dir():
-            devices.append(
-                {
-                    "path": device_path,
-                    "family": self.owproxy.read(f"{device_path}family").decode(),
-                    "type": self.owproxy.read(f"{device_path}type").decode(),
-                }
-            )
+            dtype = self.owproxy.read(f"{device_path}type").decode()
+            if dtype == MICROLAN_TYPE:
```

Comment:
Most of the code uses device family. Is there a reason why the family isn't good here?
```suggestion
            if device_family == "1F":
```